### PR TITLE
Frontend init

### LINKED ISSUE #3: front end stack initial commit

### PR DESCRIPTION
**Pull request by smknaqvi**
# Description
Adding react-gatsby app
# Closes issue(s)
closes #3 
# How to test / repro

# Screenshots

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have tested this code
- [ ] I have updated the Readme

# Other comments
todo 
[ ] add jest
[ ] add netlify deploy
[ ] run tests on pull request

### CODE REVIEW

File: frontend/src/pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from "react"
+
+const IndexPage = () => <div>Hello World {a}</div>
```
Comment (ma-anwar) [inline frontend/src/pages/index.tsx]: I'm getting an error trying to build, 

```
Building static HTML failed for path "/"

See our docs page for more info on this error: https://gatsby.dev/debug-html


  1 | import React from "react"
  2 | 
> 3 | const IndexPage = () => <div>Hello World {a}</div>
    |                         ^
  4 | 
  5 | export default IndexPage
  6 | 


  WebpackError: ReferenceError: a is not defined
  
  - index.tsx:3 
    src/pages/index.tsx:3:25
```

**Review by ma-anwar — CHANGES_REQUESTED** [review]
Will wait for further changes so I can build.

File: frontend/src/pages/index.tsx (hunk 1 above)
Comment (smknaqvi) [reply frontend/src/pages/index.tsx]: oopsies

Comment (smknaqvi) [reply frontend/src/pages/index.tsx]: > Will wait for further changes so I can build.

i ended up killing the .vscode config... turns out its no good in subdirectories, only works for whole proj so it wasn't actually doing anything heh

**Pull request merged**